QUICKSTART.md

```diff
@@ -0,0 +1,344 @@
+# **KCP-Edge** Quickstart
+
+## Required Packages:
+   - [docker](https://docs.docker.com/get-docker/)
+   - [kind](https://kind.sigs.k8s.io/)
+   - [kubectl](https://kubernetes.io/docs/tasks/tools/) (version range expected: 1.23-1.25)
+
+## Setup Instructions
+
+Table of contents:
+
+- [1. Install and run **KCP-Edge**](#1-install-kcp-edge-pre-requisites)
+- [2. Example deployment of nginx workload into two kind local clusters](#4-Example-deployment-of-nginx-workload-into-a-kind-local-cluster)
+  - [a. Stand up two kind clusters: florin and guilder](#a-Stand-up-a-local-florin-kind-cluster)
+  - [b. Onboarding the florin cluster](#b-Create-a-sync-target-placement-and-edge-syncer-for-onboarding-the-created-florin-edge-cluster)
+  - [c. Onboarding the guilder cluster](#b-Create-a-sync-target-placement-and-edge-syncer-for-onboarding-the-created-florin-edge-cluster)
+  - [d. Create and deploy the nginx workload into florin and guilder clusters](#c-Create-the-nginx-workload-and-deploy-it-to-the-florin-cluster)
+- [3. Cleanup the environment](#5-Cleanup-the-environment)
+
+
+This guide is intended to show how to quickly bring up a **KCP-Edge** environment with its dependencies from a binary release.
+
+## 1. Install and run **KCP-Edge**
+
+Download the kcp **KCP-Edge** binaries and scripts into a `kcp-edge` subfolder in your current working directory using the following command:
+
```

Comment:
The options on this script should be documented somewhere.

Since this script is primarily for use in this quickstart, I think that here is the best place for that documentation.

Reply:
OK with me to not explain here options that are not used here.

Reply:
resolving